src/PowerLincModem.ts

```diff
@@ -746,6 +746,30 @@ export default class PowerLincModem extends EventEmitter2{
 		deviceDB.devices.find(d => Number(d.cat) === cat && Number(d.subcat) === subcat);
 
 	//#endregion
+	
+	/* Send an insteon command from the modem to a device to find out what it is */
+	public queryDeviceInfo = (deviceID: Byte[]) => new Promise<Packet.StandardMessageRecieved>((resolve, reject) => {
+
+		// Catching broadcast message
+		this.once(
+			['p', PacketID.StandardMessageReceived.toString(16), MessageSubtype.BroadcastMessage.toString(16), toAddressString(deviceID)],
+			(data: Packet.StandardMessageRecieved) => {
+				if(!Array.isArray(data.to)) reject(data);
```

Comment:
Not sure why this is in here

Reply:
It's not needed. Wasn't sure at the time if `to` was always an array.